lms/static/sass/discussion/lms-discussion-main.scss

```diff
@@ -24,6 +24,14 @@ footer[role=contentinfo] {
 
 body.new-theme {
   background-color: $ash-white;
+  //sync row with UI Refresh.
+  .row{
+    margin-right: -15px;
```

Comment:
Hi @prakashpathak02 

you are chaning all .row classes in new UI and .row is part bootstrap
![image](https://user-images.githubusercontent.com/7480659/54440733-1ae11500-475d-11e9-91c8-bc51388ba08e.png)

are you sure this change is not going to create ripples?

Reply:
Murad, actually .row is being used globally in lms-discussion.css already(for older UI) so even if we don't override it globally, it may cause rows outside of discussion not work same like other tabs.  It is basically copy of UI refresh's row so it should not cause problems. We can merge and test. I have tested it on my end.

Reply:
Great I am going to merge it